controllers/sriovnetworkpoolconfig_controller_test.go

```diff
@@ -65,7 +66,7 @@ var _ = Describe("Operator", func() {
 					return err
 				}
 				return nil
-			}, timeout*3, interval).Should(Succeed())
+			}, util.APITimeout*3, util.RetryInterval).Should(Succeed())
```

Comment:
this is odd that we need it. probably related to where this test is executed.

i mean we just spin up apiserver +etcd with envtest then a simple reconcile

Reply:
IMO, that's something we have to fix in the future in a sepatare PR